recipes/pvWebMonitor/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "pvWebMonitor" %}
+{% set version = "2021.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pvWebMonitor-{{ version }}.tar.gz
+  sha256: 8d29fbe1e3415d2ca480ba9a1de6cdfc81bd53caca497d617f386ffa81e62454
+
+build:
+  entry_points:
+    - pvWebMonitor=pvWebMonitor.main:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - pyepics
+    - lxml
+    - numpy
+    - ophyd
+    - packaging
+
+test:
+  imports:
+    - pvWebMonitor
+  commands:
+    - pip check
+    - pvWebMonitor --help
+  requires:
+    - pip
+
+about:
+  home: http://bcda-aps.github.io/pvWebMonitor/
+  summary: Create static web site from EPICS PVs.
+  license: EPICS
+  license_file: src/pvWebMonitor/LICENSE
```

```suggestion
  license_file: src/pvWebMonitor/LICENSE
  dev_url: https://github.com/BCDA-APS/pvWebMonitor
  doc_url: http://bcda-aps.github.io/pvWebMonitor/
```
I've used these a few times already during review. They might be helpful in the future. The bot can point you to the dev url to inspect dependencies in the future.